src/invessel_build.py

```diff
@@ -0,0 +1,480 @@
+        build (dict): dictionary of list of toroidal and poloidal angles, as
+            well as dictionary of component names with corresponding thickness
+            matrix and optional material tag to use in H5M neutronics model.
+            The thickness matrix specifies component thickness at specified
+            (polidal angle, toroidal angle) pairs. This dictionary takes the
+            form
+            {
+                'phi_list': toroidal angles at which radial build is specified.
+                    This list should always begin at 0.0 and it is advised not
+                    to extend past one stellarator period. To build a geometry
+                    that extends beyond one period, make use of the 'repeat'
+                    parameter (list of double, deg).
+                'theta_list': poloidal angles at which radial build is
+                    specified. This list should always span 360 degrees (list
+                    of double, deg).
+                'wall_s': closed flux surface label extrapolation at wall
+                    (double),
+                'radial_build': {
+                    'component': {
+                        'thickness_matrix': list of list of double (cm),
+                        'h5m_tag': h5m_tag (str)
+                    }
+                }
+            }
+            If no alternate material tag is supplied for the H5M file, the
+            given component name will be used.
+        repeat (int): number of times to repeat build segment.
+        num_phi (int): number of phi geometric cross-sections to make for each
+            build segment (defaults to 61).
+        num_theta (int): number of points defining the geometric cross-section
+            (defaults to 61).
+        scale (double): a scaling factor between the units of VMEC and [cm]
+            (defaults to m2cm = 100).
+        plasma_h5m_tag (str): optional alternate material tag to use for
+            plasma. If none is supplied and the plasma is not excluded,
+            'plasma' will be used (defaults to None).
+        sol_h5m_tag (str): optional alternate material tag to use for
+            scrape-off layer. If none is supplied and the scrape-off layer is
+            not excluded, 'sol' will be used (defaults to None).
+        logger (object): logger object (defaults to None). If no logger is
+            supplied, a default logger will be instantiated.
+    '''
+
+    def __init__(
+            self,
+            vmec,
+            build,
+            repeat,
+            num_phi=61,
+            num_theta=61,
+            scale=m2cm,
+            plasma_h5m_tag=None,
+            sol_h5m_tag=None,
+            logger=None
+    ):
+        self.vmec = vmec
+        self.build = build
+        self.repeat = repeat
+        self.num_phi = num_phi
+        self.num_theta = num_theta
+        self.scale = scale
+        self.plasma_h5m_tag = plasma_h5m_tag
+        self.sol_h5m_tag = sol_h5m_tag
```

Comment:
Let's initialize these here

```suggestion
        self.sol_h5m_tag = sol_h5m_tag

        self.Surfaces = []
        self.Components = []

```